.circleci/config.yml

```diff
@@ -4,6 +4,11 @@ orbs:
   python: circleci/python@0.2.1
   codecov: codecov/codecov@3.2.4
 
+commands:
+  install_uv:
+    steps:
+      - run: pip install --upgrade uv">=0.5.12,<0.6.0"
```

Comment:
Given this constraint here, is it worth sticking `required-version = "<0.6.0"` in the `[uv.tool]` section of `pyproject.toml`?